src/main/java/hudson/plugins/sonar/client/SQProjectResolver.java

```diff
@@ -28,20 +29,20 @@
 
 import java.util.logging.Level;
 
-public class QualityGateResolver {
+public class SQProjectResolver {
   private final HttpClient client;
 
-  public QualityGateResolver(HttpClient client) {
+  public SQProjectResolver(HttpClient client) {
     this.client = client;
   }
-  
+
   /**
    * Resolve information concerning the quality gate.
    * Might return null if it's not possible to fetch it, which should be interpreted as 'nothing to display'.
    * Errors that should be displayed are included in {@link ProjectInformation#getErrors()}.
    */
   @CheckForNull
-  public ProjectInformation get(String projectUrl, String installationName) {
+  public ProjectInformation get(String projectUrl, String ceTaskId, String installationName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "get" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)